### DISCUSSION TITLE
Google / PayPal

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#577] [Q&A] [answered]
Hallo @devemux86 ,
ich habe heute eine Fehlermeldung von google bekommen, dass mein Cruiser ABO nicht bezahlt wurde. Ich habe jetzt noch einmal bezahlt und habe bei PayPal 2 Zahlungen für Cruiser. Kannst du mal bitte schauen, wie oft ich bezahlt habe? Gibt es eine Mailadresse von Dir, wo ich Dir ein Bildschirmfoto hinschicken kann?

Hello @devemux86 ,
I got an error message from google today that my Cruiser ABO has not been paid. I have now paid again and have 2 payments for Cruiser on PayPal. Can you please check how many times I have paid? Do you have an e-mail address where I can send you a screenshot?

**Comment by devemux86** [comment]
Hi @ToToGSARider

The email is displayed on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Google Play purchases are private, we cannot see them.
You can find your payments and Order ID here: https://support.google.com/store/answer/13714320

---

Die E-Mail wird auf [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser) angezeigt.

Google Play-Käufe sind privat, wir können sie nicht sehen.
Hier finden Sie Ihre Zahlungen und Ihre Bestell-ID: https://support.google.com/store/answer/13714320

**Reply by ToToGSARider** [reply]
@devemux86 , meinst du die Transaktions ID? 

@devemux86 , do you mean the transaction ID?

**Reply by devemux86** [reply]
@ToToGSARider 

Each Google Play purchase has a unique ID, this is how Google describes them.

Jeder Google Play-Kauf hat eine eindeutige ID, so beschreibt Google sie.

**Reply by ToToGSARider** [reply]
@devemux86 Die Bestellung hat bei google die Transaktions ID GPA.3384-0630-6414-93408..0 und ich habe eine Meldung bekommen, dass die Abbbuchung fehlgeschlagen ist. Ich habe jetzt die Zahlungsmethode aktualisiert und habe jetzt bei PayPal 2 Abbuchungen die aber angeblich beide erfolgreich abgeschlossen wurden.

@devemux86 The order has the transaction ID GPA.3384-0630-6414-93408..0 on google and I have received a message that the charge has failed. I have now updated the payment method and now have 2 debits with PayPal, both of which were supposedly completed successfully.

**Reply by ToToGSARider** [reply]
@devemux86 ich habe jetzt mal eine Mail an Google geschrieben und hoffe, dass die mir das zu viel bezahlte zurück erstatten oder geht das dann direkt bei euch auf mein Guthaben Konto? Soll ich die Mail mal hier reinsetzen? 

@devemux86 I have now written an e-mail to Google and hope that they will refund me the overpayment or will it go directly to my credit account with you? Should I put the e-mail here?

**Reply by devemux86** [reply]
@ToToGSARider 

Today there is still the order list as seen below with only 1 order.
If you don't have more order IDs, I can refund it if you want.

Heute gibt es noch die Bestellliste wie unten zu sehen mit nur 1 Bestellung.
Wenn Sie nicht mehr Auftrags-IDs haben, kann ich es erstatten, wenn Sie wollen.

https://github.com/devemux86/cruiser/discussions/577#discussioncomment-9885684

**Reply by ToToGSARider** [reply]
Okay: Hier mal ein Bild aus meinem PayPal Konto - Okay: Here is a picture from my PayPal account
![image](https://github.com/devemux86/cruiser/assets/128730865/683da282-4da7-4da9-8a99-4f8e903f1e63)

Wir warten jetzt erst noch einmal ab, ob Google antwortet auf meine Mail oder ob bei euch noch eine Buchung eingeht. 

We'll wait and see if Google responds to my email or if you receive another booking.

**Reply by devemux86** [reply]
@ToToGSARider 

What order numbers do they have? The same order numbers?

Welche Bestellnummern haben sie? Die gleichen Bestellnummern?

**Reply by devemux86** [reply]
@ToToGSARider 

You can find your orders here:

Sie können Ihre Bestellungen hier finden:

https://support.google.com/store/answer/13714320

**Reply by ToToGSARider** [reply]
Die Bezahlung vom 25.06.2024 Bestellnummer 301381173321944577 Transaktion 6U4764333B812521D
Die Bezahlung vom 26.06.2024 Bestellnummer 057219737225866395 Transaktion 5JX98583S4685645W

**Reply by devemux86** [reply]
@ToToGSARider 

We need the Google Order ID, like the previously mentioned "GPA.3384-0630-6414-93408..0" (6/25/2024).

Wir benötigen die Google Order ID, wie die bereits erwähnte "GPA.3384-0630-6414-93408..0" (6/25/2024).

https://github.com/devemux86/cruiser/discussions/577#discussioncomment-9885570

**Reply by ToToGSARider** [reply]
Bei Google ist bis jetzt nur eine Abbuchung hinterlegt für die Cruiser App für Juni 2024. Im schlimmsten Fall hat Google sich jetzt 9,99€ verdient ohne eine Leistung zu erbringen. 

So far, Google has only made a charge for the Cruiser app for June 2024. In the worst case, Google has now earned €9.99 without providing any service.

**Reply by ToToGSARider** [reply]
@devemux86 ich habe jetzt gerade mit Paypal telefoniert und die haben mir mitgeteilt, dass auf alle Fälle beide Zahlungen an den gleichen Rechnungsempfänger gegangen sein sollten. Evtl. dauert dann die Wertstellung noch etwas länger. Bitte behaltet das mal im Auge. Im Screenshot steht ja einmal Limited ausgeschrieben und einmal abgekürzt. In beiden Bestellungen steht Cruiser, sollte also beides bei euch landen. 

@devemux86 I have just phoned Paypal and they have told me that in any case both payments should have gone to the same invoice recipient. The value date may then take a little longer. Please keep an eye on this. The screenshot shows Limited written out in full and abbreviated. Both orders say Cruiser, so both should end up with you.

**Reply by ToToGSARider** [reply]
![image](https://github.com/devemux86/cruiser/assets/128730865/57e494ff-3098-4fb9-b724-b3f6900c2f41)

hier auch noch einmal die Bestätigung für die zweite Zahlung / Here is also the confirmation for the second payment

![image](https://github.com/devemux86/cruiser/assets/128730865/a3c6b7fb-5b2f-4e0d-9769-bb3451347db3)


Das nächste mal warte ich einfach ab, bis die App ggf. gesperrt wird, wenn das noch einmal passiert. Sorry für das Chaos. / Next time I'll just wait until the app is blocked if this happens again. Sorry for the mess.

**Reply by devemux86** [reply]
@ToToGSARider 

Google Play still only shows 1 order number with 1 purchase (same picture).

Google Play zeigt immer noch nur 1 Bestellnummer mit 1 Kauf an (gleiches Bild).

https://github.com/devemux86/cruiser/discussions/577#discussioncomment-9885684

**Reply by ToToGSARider** [reply]
@devemux86 wir warten jetzt einfach mal ab, was von von Google für eine Antwort kommt und schauen mal ob nächste Woche bei euch was zugebucht wurde. Hier mal Bilder von der Detailansicht:

@devemux86 We'll just wait and see what kind of response we get from Google and see if anything has been added next week. Here are some pictures of the detailed view:

![Screenshot 2024-06-27 191020](https://github.com/devemux86/cruiser/assets/128730865/8ac2418a-ad72-4588-9ef4-ca0ceb3d2508)
![Screenshot 2024-06-27 191228](https://github.com/devemux86/cruiser/assets/128730865/9c8e7ee3-6351-4f16-85be-18b68079308b)

**Reply by ToToGSARider** [reply]
@devemux86 Ich habe jetzt eine Antwort vom Google Support: Hallo Herr XXX,
vielen Dank für Ihre E-Mail.
Ich habe eine Antwort von unserem Produktexperten erhalten. Hier ist die Erklärung zu den 9,99 Euro. Bei den 9,99 Euro handelt es sich um eine Autorisierung. Autorisierungen sind die erste Kommunikation zwischen dem Abrechnungssystem von Google und dem Abrechnungssystem der Bank des Kunden, um sicherzustellen, dass die Zahlungsmethode gültig ist und die erforderlichen Mittel vorhanden sind. Autorisierungen belasten Ihr Konto nicht. Wenn eine Autorisierung erfolgt, wird kein Geld von Ihnen abgezogen, auch wenn der Betrag vorübergehend auf Ihrem Kontoauszug oder Ihrer Transaktionsliste erscheint.
Diese ausstehenden Autorisierungen bleiben normalerweise 1-14 Werktage auf dem Bankkonto des Kunden. Wie lange dies dauert, hängt ganz von der Bank des Kunden ab. Sobald die Bank ihren Vorgang abgeschlossen hat, verschwindet die Autorisierung vom Kontoauszug des Kunden.
Wenn Sie etwas im Google Store / bei Google Play kaufen, wird eine vorübergehende Vorautorisierung auf Ihre Zahlungsmethode gesetzt, um zu bestätigen, dass ausreichende Mittel vorhanden sind. Diese Vorautorisierung wird widerrufen, sobald die tatsächliche Abbuchung bei Versand der Bestellung erfolgt. Wenn auf Ihrem Bankauszug eine doppelte Belastung angezeigt wird, kann es sich sowohl um die Vorautorisierung als auch um die tatsächliche Belastung handeln.

Ich wünsche Ihnen weiterhin einen schönen Tag!

@devemux86 I now have an answer from Google Support: Hello Mr. XXX,
Thank you for your email.
I have received a reply from our product expert. Here is the explanation of the 9.99 euros. The 9.99 euros is an authorization. Authorizations are the first communication between Google's billing system and the customer's bank's billing system to ensure that the payment method is valid and the required funds are available. Authorizations do not debit your account. When an authorization is made, no money will be deducted from you, even if the amount appears temporarily on your statement or transaction list.
These pending authorizations usually remain in the customer's bank account for 1-14 business days. How long this takes depends entirely on the customer's bank. Once the bank has completed its process, the authorization disappears from the customer's bank statement.
When you purchase something from the Google Store / Google Play, a temporary pre-authorization will be placed on your payment method to confirm that sufficient funds are available. This pre-authorization will be revoked as soon as the actual charge is made when the order is shipped. If you see a duplicate debit on your bank statement, it may be both the pre-authorization and the actual debit.

Have a nice day!
*** Translated with www.DeepL.com/Translator (free version) ***

**Reply by devemux86** [reply]
@ToToGSARider 

Thanks for providing the answer from Google Support!

Vielen Dank für das Verfassen der Antwort vom Google-Support!

**Reply by ToToGSARider** [reply, marked as answer]
@devemux86 Ich habe den / die Schuldige gefunden. Meine Frau hat ja auch die Cruiser App auf Ihrem Handy laufen mit dem gleichen PayPal Account. Wir haben also auf 2 Handys mit unterschiedlichen Google Accounts, jedoch mit den gleichen PayPal Accounts, die App laufen. Dies könnte evtl. die Meldung verursacht haben. Ist ja dann aber auch Okay, da wir ja 2 Lizenzen benötigen. Hatte ich vergessen zu kontrollieren. Sorry, und einen schönen Sonntag noch.

@devemux86 I have found the culprit. My wife also has the Cruiser app running on her cell phone with the same PayPal account. So we have the app running on 2 cell phones with different Google accounts, but with the same PayPal accounts. This could possibly have caused the message. But that's okay, because we need 2 licenses. I had forgotten to check. Sorry, and have a nice Sunday.

**Reply by devemux86** [reply]
@ToToGSARider 

Thank you for the information! Then you should have 2 different order numbers.

Vielen Dank für die Information! Dann sollten Sie 2 verschiedene Bestellnummern haben.

**Comment by devemux86** [comment]
@ToToGSARider 

This is the history of orders for this ID (without private data). There is only 1 payment on 6/25/2024.
Do you have another Order ID? If we refund this order, it could cancel your subscription?

Dies ist die Historie der Bestellungen für diese ID (ohne private Daten). Es gibt nur 1 Zahlung am 6/25/2024.
Haben Sie eine andere Bestell-ID? Wenn wir diese Bestellung zurückerstatten, könnte das Ihr Abonnement beenden?

![Orders](https://github.com/devemux86/cruiser/assets/3484020/7d59d94c-8521-40ca-ae2e-a1040003e089)

**Reply by ToToGSARider** [reply]
@devemux86 Nein, bisher nicht. Seltsam ist jetzt nur, dass bei euch die Zahlung gestern schon eingegangen sein soll. Bitte behaltet es mal im Auge. Ich achte bei mir auch mal auf die Konten was da passiert. Wenn nur eine Zahlung bei euch eingegangen ist, ist ja alles gut.

@devemux86 No, not yet. The only strange thing is that the payment should have been received yesterday. Please keep an eye on it. I'll also keep an eye on my accounts to see what's happening. If you have only received one payment, then all is well.

**Reply by ToToGSARider** [reply]
Das habe ich per Mail von Google erhalten:
Deine Zahlung von PayPal wurde abgelehnt.

Wenn du deine Zahlungsmethode nicht aktualisierst, verlierst du alle Vorteile von „Cruiser Pro (Cruiser)“. Zu diesen Vorteilen gehört Folgendes:

Sprachgeführte Navigation

GPS-Tracks aufzeichnen

Viele Konfigurationsoptionen

Keine Werbung

Damit du diese Vorteile weiter nutzen kannst, muss dein Abo aktiv bleiben. Aktualisiere deshalb deine Zahlungsmethode oder verwende eine andere.

I received this by email from Google:
Your PayPal payment has been declined.

If you do not update your payment method, you will lose all the benefits of "Cruiser Pro (Cruiser)". These benefits include the following:

Voice-guided navigation

Record GPS tracks

Many configuration options

No advertising

Your subscription must remain active for you to continue to enjoy these benefits. Therefore, update your payment method or use a different one.

**Reply by devemux86** [reply]
@ToToGSARider 

No problem, if you need something else or have more than one order number, please let me know.

Kein Problem, wenn Sie etwas anderes benötigen oder mehr als eine Bestellnummer haben, lassen Sie es mich bitte wissen.

**Reply by ToToGSARider** [reply]
@devemux86 ich habe jetzt auf dem 2.Konto, wo die Zahlung angeblich fehlgeschlagen ist eine Abbuchung. Bitte schaut morgen mal nach, ob da noch eine Zahlung eingeht.

@devemux86 I now have a debit on the 2nd account where the payment supposedly failed. Please check tomorrow to see if another payment has been received.

**Comment by WolfgangZ42** [comment]
**Google seems to have problems with the administration of subscriptions ... 
Some time ago I received an eMAIL with the announcement of a payment on July 06, 2024 - which definitely cannot be true:** 

![2024-06-26 20_26_20-#open - wstoehler@gmail com - Outlook](https://github.com/devemux86/cruiser/assets/121806238/6c5436b6-5089-4ccf-8225-cbafcd8db5d5)

**I'll wait and see if there really is a direct debit.**

**My subscription status is as follows:**
![2024-06-26 20_30_16-Google Play](https://github.com/devemux86/cruiser/assets/121806238/07584e59-5ccb-44bc-9e22-de43b657565d)

**And the order history:** 
![2024-06-26 20_28_43-Bestellverlauf – Google Play](https://github.com/devemux86/cruiser/assets/121806238/68eb54d6-e2aa-4bc8-8a3d-edbae5797476)

**_Also actually quite manageable - as I said, I will only react after an unauthorized payment._**

**Reply by devemux86** [reply]
@WolfgangZ42 

Do you mean they write July when it should be renewed on November?

**Reply by ToToGSARider** [reply]
Also ich habe laut Google um 09:10 Uhr meine Cruiser App bezahlt. Würde ja passen mit Zeitzone, ich habe bei Paypal jetzt gesagt Guthaben nutzen.
Auf dem Konto was ich normalerweise bei Paypal verwende, habe ich für morgen eine Wertstellung bzgl. einer Google Zahlung. Vermutlich habe ich dann in den nächsten Tagen 2 Zahlungen bei Google drin.

So I paid for my Cruiser app at 09:10 according to Google. Would fit with time zone, I have now said to use credit with Paypal.
On the account I normally use with Paypal, I have a value date for tomorrow regarding a Google payment. I will probably have 2 payments with Google in the next few days.

**Reply by WolfgangZ42** [reply]
> Do you mean they write July when it should be renewed on November?

exactly!

**Reply by WolfgangZ42** [reply]
Okay - my fault!
In the direct debit, the sub-google account was then correctly specified. I had set up a spare smartphone for fellow riders with an additional google account (so that neither postbox nor payment methods can be accessed). Thus, I pay twice for Cruiser - and it's worth every penny!

**Reply by devemux86** [reply]
@WolfgangZ42 thanks for the clarification!
